src/Jeffijoe.MessageFormat/MessageFormatter.cs

```diff
@@ -232,10 +232,11 @@ public string FormatMessage(string pattern, IDictionary<string, object> args)
             var requests = this.ParseRequests(pattern, sourceBuilder);
             var requestsEnumerated = requests.ToArray();
 
-            // If we got no formatters, then we're done here.
+            // If we got no formatters, then we only need to unescape the literals.
             if (requestsEnumerated.Length == 0)
             {
-                return pattern;
+                sourceBuilder = this.UnescapeLiterals(sourceBuilder);
```

Comment:
This fixes the problem that currently "Hello \{World\}" results in "Hello \{World\}" instead of "Hello {World}". If my PR gets rejected, then at least this should be merged.